package.json

```diff
@@ -22,6 +22,7 @@
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
     "lint": "turbo lint",
     "prompts": "turbo prompts --filter=core",
+    "prompts:dev": "turbo prompts:dev --filter=core",
```

Comment:
When you run this it should fill up your prompts table with all prompts and variants